tools/PublishModules.ps1

```diff
@@ -174,10 +179,20 @@ function Change-RMModule
 		  Write-Output "Expanding $zipPath"
 		  Expand-Archive $zipPath
 		  Write-Output "Adding PSM1 dependency to $unzippedManifest"
-		  Add-PSM1Dependency -Path $unzippedManifest
-		  Write-Output "Removing module manifest dependencies for $unzippedManifest"
-		  Remove-ModuleDependencies -Path $unzippedManifest
 
+          if ($moduleName -ne "AzureStack")
```

Comment:
@markcowl The function change-rmmodule in addition to adding psm1 dependency, it also packages and publishes the module to local temp repo. So the function needs to be called for azurestack as well. could you please check?